index.html

```diff
@@ -289,9 +289,18 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
-      <div class="contact">
-        <form action=""></form>
-      </div>
+      <section class="contact">
+        <h2>I Would love to here from you. If you have a project or a feature
+          that you would like to discuss, please get in touch giving some summary 
+          using this form.
+        </h2>
+        <form action="https://formspree.io/f/myylpdqq" method="POST">
+          <input type="text" name="full-name" placeholder="Name">
+          <input type="email" name="email" placeholder="Email address" required>
+          <input type="text" maxlength="500" placeholder="message..." required>
+          <button type="submit" class="form-submit">Get in touch</button>
```

Comment:
Please prevent your user to send the form if one or more fields are empty.
As a quick reminder, here is the screenshot of the requirements:
![requirements](https://user-images.githubusercontent.com/47100064/124171211-ed950e00-dab0-11eb-9070-90247870db3b.png)